XiEditor/Search.swift

```diff
@@ -138,7 +138,7 @@ class FindViewController: NSViewController, NSSearchFieldDelegate, NSControlText
         if NSEvent.modifierFlags.contains(.shift) {
             findDelegate.findPrevious(wrapAround: wrapAround)
         } else {
-            findDelegate.findNext(wrapAround: wrapAround, allowSame: false)
```

Comment:
is this change here intentionally? It does not seem like it should be part of this PR.

Reply:
Oh, I think I just tried something out and forgot to change it back. I will undo it.